### ISSUE TITLE
set_name: Selector-Error: Invalid selection name

### ISSUE THREAD

**Issue by sbwiecko** [schrodinger/pymol-open-source#251]
once I have selected a few residues, I tried to rename the selection using the command line
`set_name sele,new_name`
but I receive an error message
```
Selector-Error: Invalid selection name "new_name".
new_name<--
```
At the end, the selection has been renamed properly, and there is no message error when I rename the selection using the menu [A]>Rename Selection.

**Comment by JarrettSJohnson** [comment]
I can reproduce with non-Open Source version. Open Source version seems fine. Can you confirm?

**Comment by sbwiecko** [comment]
I confirm, no problem using the version 2.5.0 Open-Source version of PyMol, but PyMOL 2.5.4 - Incentive Version drops the error message while renaming a selection on the command line 👍

**Comment by JarrettSJohnson** [comment]
Thanks for confirming. I'll close this issue for now since this is the Open Source issue tracker, but I have added it as a bug internally and will have this addressed for the 2.5.5 release.
